[WIP] ROSA topic map restructure (#41923)
openshift/openshift-docs: rosa_cluster_admin/rosa_monitoring/rosa-troubleshooting-monitoring-issues.adoc

:_content-type: ASSEMBLY
[id="rosa-troubleshooting-monitoring-issues"]
= Troubleshooting monitoring issues
include::_attributes/attributes-openshift-dedicated.adoc[]
:context: rosa-troubleshooting-monitoring-issues

toc::[]

> This document describes how to troubleshoot common monitoring issues for user-defined projects.

Reviewer: Maybe:

```suggestion
Find troubleshooting steps for common monitoring issues with user-defined projects.
```
